---a/net/minecraft/world/biome/BiomeGenBeach.java
+++b/net/minecraft/world/biome/BiomeGenBeach.java
@@ -2,12 +2,12 @@
 
 import net.minecraft.init.Blocks;
 
-public class BiomeGenBeach extends BiomeGenBase
-{
+public class BiomeGenBeach extends BiomeGenBase {
+
     private static final String __OBFID = "CL_00000157";
 
-    public BiomeGenBeach(int p_i1969_1_)
-    {
+    public BiomeGenBeach(int p_i1969_1_) {
         super(p_i1969_1_);
         this.spawnableCreatureList.clear();
         this.topBlock = Blocks.sand.getDefaultState();
@@ -17,4 +17,4 @@
         this.theBiomeDecorator.reedsPerChunk = 0;
         this.theBiomeDecorator.cactiPerChunk = 0;
     }
-}
+}
